guides/release/pages.yml

```
@@ -1,325 +1,325 @@
           url: "index"
-        - title: "Route Params"
+        - title: "Paramètres des Routes"
           url: "route-params"
-        - title: "Service Injection"
+        - title: "Injection de Service"
           url: "service-injection"
         - title: "Ember Data"
           url: "ember-data"
-        - title: "Provider Components"
+        - title: "Composants Provider"
           url: "provider-components"
-        - title: "Recap"
+        - title: "Récap"
           url: "recap"
 
 - title: "Core Concepts"
   url: "toc-heading_core-concepts"
   is_heading: true
-- title: "Components"
+- title: "Composants"
   url: components
   pages:
-    - title: "Templates are HTML"
+    - title: "Les Templates sont du HTML"
       url: "index"
-    - title: "Introducing Components"
+    - title: "Introduction aux Composants"
       url: "introducing-components"
-    - title: "Component Arguments and HTML Attributes"
+    - title: "Arguments des Composants et Attributs HTML"
       url: "component-arguments-and-html-attributes"
-    - title: "Conditional Content"
+    - title: "Contenu Conditionnel"
       url: "conditional-content"
-    - title: "Block Content"
+    - title: "Contenu des Blocs"
       url: "block-content"
-    - title: "Helper Functions"
+    - title: "Fonctions Helper"
       url: "helper-functions"
-    - title: "Component State and Actions"
+    - title: "État des Composants et Actions"
       url: "component-state-and-actions"
-    - title: "Looping Through Lists"
+    - title: "Boucler sur des Listes"
       url: "looping-through-lists"
-    - title: Template Lifecycle, DOM, and Modifiers
+    - title: Cycle de Vie des Templates, DOM, et Modifiers
       url: template-lifecycle-dom-and-modifiers
-    - title: "Built-in Components"
+    - title: "Composants Built-in"
       url: "built-in-components"
 - title: "Routing"
   url: "routing"
   pages:
     - title: "Introduction"
       url: "index"
-    - title: "Defining Your Routes"
+    - title: "Définir une Route"
       url: "defining-your-routes"
-    - title: "Linking Between Routes"
+    - title: "Liens entre les Routes"
       url: "linking-between-routes"
-    - title: "Specifying a Route's Model"
+    - title: "Spécifier le Modèle d'une Route"
       url: "specifying-a-routes-model"
-    - title: "Rendering a Template"
+    - title: "Rendre un Template"
       url: "rendering-a-template"
-    - title: "Redirecting"
+    - title: "Redirections"
       url: "redirection"
-    - title: "Preventing and Retrying Transitions"
+    - title: "Empêcher et Réessayer une Transition"
       url: "preventing-and-retrying-transitions"
       isAdvanced: true
-    - title: "Loading / Error Substates"
+    - title: "Chargement / Sous-États d'Erreur"
       url: "loading-and-error-substates"
       isAdvanced: true
-    - title: "Query Parameters"
+    - title: "Query Params"
       url: "query-params"
-    - title: "Asynchronous Routing"
+    - title: "Routing Asynchrone"
```

`Routage` is a thing, even if it sounds weird.

```suggestion
    - title: "Routage Asynchrone"
```

or maybe we could slightly differ?

```suggestion
    - title: "Les Routes Sont Asynchrones"
```

```suggestion
    - title: "Routes Asynchrones"
```